### DISCUSSION TITLE
SOC appeared once then IP times out everytime ....

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5620] [Unsupported Versions]
Hi, I installed "2.3.61-MSEARCH " on VMware workstation 16 pro. 
  I used this video "https://www.youtube.com/watch?v=TG-pPm7KVro  ". 
 It had me using DHCP, standalone, ens33 for management, direct connect to the internet, ens34 for a monitor

  Could i have lost the DHCP IP config? I do have a snapshot.
  The installation ended up with "install had a problem". 
![image](https://user-images.githubusercontent.com/5569263/134269066-78be50da-0aff-43b3-bce3-77acf02cd8b9.png)
when i logged back in, i saw this 
![image](https://user-images.githubusercontent.com/5569263/134269148-60caaa49-b79a-4546-87a9-b7d03604c63a.png)
then i tried to log in using chrome on my host desktop and saw this eventually:
![image](https://user-images.githubusercontent.com/5569263/134269305-59bebe97-0846-45ee-9d20-ceb0b0b6b1ef.png)
  I successfully logged in and it showed my user email address. But now, i cannot get the SOC again? When I try I see 
![image](https://user-images.githubusercontent.com/5569263/134269509-e156a9f8-b7d1-4cb9-98f3-ac72cd495f98.png)

**Comment by Sahale** [comment]
What's in error log?
If you use DHCP, IP can change. SO recommends to use static IP (or you can use DHCP reservation at least). Shapshot can help you but it's not recommended because it can lead to IP conflict.
